apps/api/src/app/workflows-v2/usecases/delete-workflow/delete-workflow.command.ts

```diff
@@ -0,0 +1,10 @@
+import { EnvironmentWithUserObjectCommand } from '@novu/application-generic';
+import { IsDefined, IsString } from 'class-validator';
+
+export type RequiredProp<T, K extends keyof T> = T & { [P in K]-?: T[P] };
```

Comment:
delete

Reply:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** This generic utility type is declared elsewhere in the `api` application. Please remove one of them to avoid duplication.‏

Reply:
deleted